vector/src/actcwd/res/values/config.xml

```diff
@@ -55,7 +55,7 @@
     <bool name="settings_show_timestamp_visible">false</bool>
     <bool name="settings_show_read_receipt_visible">false</bool>
     <bool name="settings_flair_visible">false</bool>
-    <bool name="settings_use_native_camera_visible">true</bool>
+    <bool name="settings_use_native_camera_visible">false</bool>
```

Comment:
Considering this was set to true to fix: https://dev.azure.com/ACTGovernment/Pegacorn/_workitems/edit/190948 should the Media setting for Use Native Camera be set to false and hidden/not changable to avoid https://dev.azure.com/ACTGovernment/Pegacorn/_workitems/edit/190425

Reply:
"settings_use_native_camera_visible" is whether we show or hide the settings item. Currently, its hidden both in CWD and BYOD.  "settings_native_camera_use_default_value" is used which camera the app should use by default (does not matter whether the settings is shown or not), false = mobile camera and true = app camera.

Reply:
Sorry my mistake, thought they were the same config item